VTEX - Ad Network API.json

```diff
@@ -608,6 +617,399 @@
+                                        "example": "tshirt XYZ"
+                                    },
+                                    "campaignId": {
+                                        "type": "string",
+                                        "description": "Campaign ID to which the ad belongs.",
+                                        "example": "9e424dcd-9b46-456e-a3a5-05803c83452f"
+                                    },
+                                    "adId": {
+                                        "type": "string",
+                                        "description": "Ad identifier associated with the ad event. This ID should be retrieved through the [ad request](https://developers.vtex.com/docs/api-reference/vtex-ad-network-api#get-/sponsored_products/-facets-).",
+                                        "example": "05949176-3a10-435e-9d08-3ba8da51de3b"
+                                    },
+                                    "channel": {
+                                        "type": "string",
+                                        "description": "Specifies if it is a mobile or web client.",
+                                        "enum": [
+                                            "website",
+                                            "android",
+                                            "ios",
+                                            "msite",
+                                            "whatsapp"
+                                        ],
+                                        "example": "website"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/order": {
+            "servers": [
+                {
+                    "url": "https://af-origin.vtex.com/api/activity-flow",
+                    "description": "Ad Network server."
+                }
+            ],
+            "post": {
+                "summary": "Send order events",
+                "description": "Sends order events.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+                "tags": [
+                    "Order events"
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "description": "Order event request body.",
```

Comment:
There is no need to add a description to the request body, our guidelines do not require it.